assets/javascripts/modules/ajaxCallbacks.js

```diff
@@ -96,6 +96,9 @@ var ajaxCallbacks = {
           $parent.append($offDisabled);
         }
 
+        // update no of subscriptions
+        $('[data-api-subscriptions="'+ response.apiName +'"]').text(response.numberOfSubscriptionText);
+
```

Comment:
> can this be done in a more generic fashion? the use of the attribute data-api-subscriptions sounds service specific

my bad this is already in a service specific `apiSubscribeResponse` function! We should look at the ticket to make this work generic and get it in play